src/test/java/features/CreateEvent.feature

```diff
@@ -0,0 +1,42 @@
+@CreatingDifferentTypesOfEvents
+Feature: Create events
+  This feature deals with the create events functionality of the application
+
+  Background: I log in to Google
+    Given I am logged in
+    Then I should see the Create Event button
+
+  @current
+  Scenario Outline: Creating multiple events
+    When I click on Create button
+    Then The creating new event page should be displayed
```

Comment:
The beginning character of the step definitions are usually lowercase, except nouns written uppercase, like "I" or "Empire State Building", as the Given, When, And, Then keywords represent the beginning of the sentences in feature files.